components/activity/params.go

```diff
@@ -9,7 +9,7 @@ import (
 // ParametersActivity contains the definition of the configuration parameters used by the Activity component.
 type ParametersActivity struct {
 	// Enabled defines whether the Activity component is enabled.
-	Enabled bool `default:"true" usage:"whether the Activity component is enabled"`
+	Enabled bool `default:"false" usage:"whether the Activity component is enabled"`
```

Comment:
Do we still need the activity component then? I think you can remove it

Reply:
Removed.